### ISSUE TITLE
Screenshots out of date

### ISSUE THREAD

**Issue by Brontojoris** [AdobeDocs/analytics.en#219]
These screenshots are out of date, making it hard to follow along with the instructions.

Issue in ./help/components/c-calcmetrics/c-workflow/cm-workflow/c-build-metrics/cm-weighted-metric.md

REF: https://experienceleague.adobe.com/docs/analytics/components/calculated-metrics/calcmetric-workflow/cm-weighted-metric.html?lang=en

**Comment by MicaPete** [comment]
@Brontojoris - I have updated the screenshots.
https://experienceleague.adobe.com/docs/analytics/components/calculated-metrics/calcmetric-workflow/cm-weighted-metric.html?lang=en
